setup.py

```diff
@@ -21,6 +21,7 @@
         "cytoolz>=0.9.0,<1.0.0;implementation_name=='cpython'",
         "eth-abi>=1.1.1,<2",
         "eth-account>=0.2.1,<0.3.0",
+        "eth-pm>=0.1.0-beta.17,<1.0.0",
```

Comment:
I'd pin to a specific beta unless you're confident that breaking changes won't be introduced in subsequent betas.